### ISSUE TITLE
Is your fine-tuned model based on the pre-trained model trained with WSD

### ISSUE THREAD

**Issue by adol001** [bilibili/Index-1.9B#36]
Hope to get a reply.

**Comment by mayokaze** [comment]
Yes, both Index-1.9B-Chat and Index-1.9B-32K are based on  Index-1.9B which was pre-trained with WSD scheduler.
